src/v2/summary.ts

```diff
@@ -0,0 +1,32 @@
+import { getAddress } from '@ethersproject/address'
+import { NowRequest, NowResponse } from '@now/node'
+
+import { return200, return500 } from '../utils'
+import { getTopPairs } from './_shared'
+
+interface ReturnShape {
+  [tokenIds: string]: { last_price: string; base_volume: string; quote_volume: string }
+}
+
+export default async function(req: NowRequest, res: NowResponse): Promise<void> {
+  try {
+    const pairs = await getTopPairs(true)
+
+    return200(
+      res,
+      pairs.reduce<ReturnShape>((accumulator, pair, i): any => {
+        const id0 = getAddress(pair.token0.id)
+        const id1 = getAddress(pair.token1.id)
+        accumulator[`${id0}_${id1}`] = {
+          last_price: pair.token0Price,
+          base_volume: pair.volumeToken0,
+          quote_volume: pair.volumeToken1
```

Comment:
unless i'm mistaken, this is all-time volume, and i believe the CMC spec asks for 24 hr?